test/components/previews/card_component_preview.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class CardComponentPreview < ViewComponent::Preview
+  def default
+    render Viral::CardComponent.new(title: 'Simple card with Title') do
+      'This is a card'
+    end
+  end
+
+  def card_with_header_actions
+    render Viral::CardComponent.new(title: 'Card with header actions') do |card|
```

Comment:
For some reason this preview uses a different font size for the header text, can you see why this is happening.

Reply:
Thanks, updated in bd31499